README.md

```diff
@@ -253,6 +253,12 @@ Use the following parameters to customize the `Tasks`.
 | `cpu-count` | The number of CPU cores to dedicate for the VM. Must be 3, 4, 6, 8, 12, or 24. | 3 |
 | `vcpu-count` | The number of vCPUs for the VM. Must equal the number of CPUs, when CPU is less than or equal to 3. Otherwise, must equal half of or exactly the number of CPUs specified. | 3 |
 | `vnc-console` | Enables or disables VNC for the VM. | false |
+| `vm-metadata` | Inject custom metadata to the VM. You need to provide the metadata in format: `items: [{ key: testKey, value: testValue }]`. | --- |
```

Comment:
It is worth mentioning here as well that vm metadata is (currently) for Intel only

Reply:
oops, forgot that